osf_tests/factories.py

```diff
@@ -580,18 +580,25 @@ def sync_set_identifiers(preprint):
 
 class PreprintFactory(DjangoModelFactory):
```

Comment:
Main change to PreprintFactory is that a node isn't required (this helped catch a ton more things in tests). You can optionally add one though.